teuthology/run_tasks.py

```diff
@@ -141,6 +141,7 @@ def run_tasks(tasks, ctx):
 
         if ctx.config.get('interactive-on-error'):
             ctx.config['interactive-on-error'] = False
+            del ctx.config['archive_path']
```

Comment:
Why this is required here?

Reply:
I observed that archive_path used to hider with manual interactive on error method when using orig.config.yaml of a teuthology job, but you are right I can verify it's not required and call runs smooth without this configuration as well, removed